llvm/lib/Transforms/IPO/FunctionAttrs.cpp

```diff
@@ -128,6 +128,15 @@ static void addLocAccess(MemoryEffects &ME, const MemoryLocation &Loc,
     ME |= MemoryEffects::argMemOnly(MR);
     return;
   }
+  // TODO: This should be refined to use upcoming Loc.TBAAErrno for errno
+  // memory, rather than manually inspecting the underlying object.
+  if (isa<CallInst>(UO)) {
+    auto *Callee = cast<CallInst>(UO)->getCalledFunction();
+    if (Callee && Callee->getName() == "__errno_location") {
```

Comment:
We use `_errno` on Windows: https://godbolt.org/z/T6oanEKKa
See also https://github.com/llvm/llvm-project/blob/fdadef9be33fb3f0844b85c35a63a8a4137382dd/clang/lib/StaticAnalyzer/Checkers/ErrnoModeling.cpp#L43-L50.

Reply:
Right, clang on Windows links against UCRT, which declares _errno. This has to be taken into account.